userspace/falco/config_json_schema.h

```diff
@@ -38,7 +38,30 @@ const char config_schema_string[] = LONG_STRING_CONST(
                 "config_files": {
                     "type": "array",
                     "items": {
-                        "type": "string"
+                        "oneOf": [
```

Comment:
Either string or an object.
In case of object, `path` is required, while `strategy` must be an enum between `append,override,add-only`.